transformer_engine/common/__init__.py

```diff
@@ -82,7 +82,19 @@ def _load_cudnn():
 def _load_library():
     """Load shared library with Transformer Engine C extensions"""
 
-    so_path = get_te_path() / "transformer_engine" / f"libtransformer_engine.{_get_sys_extension()}"
+    so_path = (
+        get_te_path()
+        / "transformer_engine"
+        / "wheel_lib"
+        / f"libtransformer_engine.{_get_sys_extension()}"
+    )
+    if not so_path.exists():
```

Comment:
I understand that we are counting on both of these never being valid at the same time, but I would still like these lines to be swapped here (so source has precedence if it exists).

Reply:
Done